subworkflows/local/depths.nf

```diff
@@ -53,9 +61,10 @@ workflow DEPTHS {
         .collectFile(name:'sample_groups.tsv'){ meta, reads -> meta.id + '\t' + meta.group + '\n' }
 
     // Filter MAG depth files: use only those for plotting that contain depths for > 2 samples
+    // as well as > 2 bins
     ch_mag_depths_plot = MAG_DEPTHS.out.depths
         .map { meta, bin_depths_file ->
-            if (getColNo(bin_depths_file) > 2) [ meta, bin_depths_file ]
+            if (getColNo(bin_depths_file) > 2 && getRowNo(bin_depths_file) > 2) [ meta, bin_depths_file ]
```

Comment:
This appears to read `bin_depths_file` twice.  If that file is expected to ever be large, may want to implement both column and row filters in the same method.

Reply:
Thanks @heuermh ! Good point, Generally it shouldn't be big, but will keep that in mind for the future :)